[fix] index.d.ts missing in package.json of ecs-morgan-format

Hi, 
I tried using the @elastic/ecs-morgan-format in my ts project which gave following error,
`Cannot find module '@elastic/ecs-morgan-format' or its corresponding type declarations.`

After looking in the code I found `index.d.ts` is missing in the files property of [package.json](https://github.com/elastic/ecs-logging-nodejs/blob/23c6d6ec8e26e5396f44e4f79ae9b79504582bae/packages/ecs-morgan-format/package.json#L8). 


Creating PR to solve this small issue. ✌️

--- a/packages/ecs-morgan-format/package.json
+++ b/packages/ecs-morgan-format/package.json
@@ -5,7 +5,8 @@
   "main": "index.js",
   "types": "index.d.ts",
   "files": [
-    "index.js"
+    "index.js",
+    "index.d.ts"
   ],
   "repository": {
     "type": "git",

